### PR TITLE
Wqp 1409 cleanup

### PR DESCRIPTION
**Pull request by mbucknell**
Changes required to get the ETL to run to completion. There is likely more that can and should be done to improve performance.

### CODE REVIEW

File: src/main/java/gov/acwi/wqp/etl/NwisJdbcPagingItemReader.java [hunk 1]
```diff
@@ -0,0 +1,295 @@
+package gov.acwi.wqp.etl;
```
Comment (dsteinich) [inline src/main/java/gov/acwi/wqp/etl/NwisJdbcPagingItemReader.java]: Possibly better to extend JdbcPagingItemReader than re-implement it. It's not obvious what needed to be changed just by looking at this source.

File: src/main/java/gov/acwi/wqp/etl/result/TransformResult.java [hunk 2]
```diff
@@ -52,18 +56,33 @@
...
-		.rowMapper(new NwisResultRowMapper())
-		.build();
+	public NwisJdbcPagingItemReader<NwisResult> resultReader() throws Exception{
+		NwisPostgresPagingQueryProvider queryProvider = new NwisPostgresPagingQueryProvider();
+		Map<String, Order> sortKeys = new LinkedHashMap<>();
+
+		queryProvider.setSelectClause(new String(FileCopyUtils.copyToByteArray(selectClause.getInputStream())));
+		queryProvider.setFromClause(new String(FileCopyUtils.copyToByteArray(fromClause.getInputStream())));
+		queryProvider.setWhereClause(new String(FileCopyUtils.copyToByteArray(whereClause.getInputStream())));
+		sortKeys.put("qw_result.result_id", Order.ASCENDING);
```
Comment (dsteinich) [inline src/main/java/gov/acwi/wqp/etl/result/TransformResult.java]: Can this use the regular JdbcPagingItemReader now?

File: src/main/java/gov/acwi/wqp/etl/NwisJdbcPagingItemReader.java (hunk 1 above)
Comment (mbucknell) [reply src/main/java/gov/acwi/wqp/etl/NwisJdbcPagingItemReader.java]: I agree. The part that changed was using getSortKeyWithoutAlias. It's within a private function that is then used with the RowMapper implementation. It was expedient at the time to just copy the whole thing then figure out what exactly needed to be reimplemented.

At this point, I think I'll put in a //TODO to do this at a later time (if we end up keeping it).

File: src/main/java/gov/acwi/wqp/etl/result/TransformResult.java (hunk 2 above)
Comment (mbucknell) [reply src/main/java/gov/acwi/wqp/etl/result/TransformResult.java]: I don't think so because of the joins. I can test locally though.

Comment (mbucknell) [reply src/main/java/gov/acwi/wqp/etl/result/TransformResult.java]: The answer is no

**Pull request merged**